drv/lpc55-sprot-server/src/main.rs

```diff
@@ -227,7 +237,7 @@ impl IO {
     /// Returns false on spurious interrupt.
     fn pio(&mut self, transmit: bool) {
         ringbuf_entry!(Trace::Pio(transmit));
-        let tx_end = self.tx_buf.len(); // Available bytes and trailing zeros
+        let tx_end = self.tx_buf.as_slice().len(); // Available bytes and trailing zeros
```

Comment:
Why does `RxBuf` have a `.len()` function, but `TxBuf` does not?

Reply:
Because I needed to get the length of the inner buffer for `RxBuf` :) These types definitely aren't symmetrical, but maybe they should be.